metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -64,16 +68,26 @@ function remote_network_bandwidth_iperf3 {
 	start_server
 
 	client_id=$($DOCKER_EXE ps -q)
-	client_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c $server_ip_address -t $server_time"
-	result=$(start_client "$client_id" "$client_command")
-	total_bandwidth=$(echo "$result" | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}')
-	echo "Network bandwidth is : $total_bandwidth"
-
+	if [ "$runtime_client" == "runc" ]; then
```

Comment:
there is a lot of repetition here, both for the runtime check code and the iperf invocation code - it would be nice if we could factor those out into one or two more generic functions - but, if we can't then fine.